beprof/curve.py

```diff
@@ -31,8 +31,12 @@ class Curve(np.ndarray):
     Has additional field - info.
     """
 
-    def __new__(cls, input_array):
+    def __new__(cls, input_array, **meta):
         obj = np.asarray(input_array).view(cls).copy()
```

Comment:
Works without .copy() but that was something you wrote so I didn't dare to remove it!